### PR TITLE
Als 3072 ancillary apps aws accounts

### PR DESCRIPTION
**Pull request by smjmoj**

### CODE REVIEW

File: .github/workflows/tag-master-branch-on-merge.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Bump version
+on:
+  push:
+    branches:
+      - master
```
Comment (pasipa2) [inline .github/workflows/tag-master-branch-on-merge.yaml:5]: Could we change to use a different branch like main etc

File: internetgateway/terragrunt.hcl [hunk 2]
```diff
@@ -0,0 +1,7 @@
+include {
+  path = find_in_parent_folders()
+}
+
+dependencies {
+  paths = ["../vpc"]
+}
```
Comment (pasipa2) [inline internetgateway/terragrunt.hcl:7]: blank line

File: natgateway/terragrunt.hcl [hunk 3]
```diff
@@ -0,0 +1,7 @@
+include {
+  path = find_in_parent_folders()
+}
+
+dependencies {
+  paths = ["../vpc", "../internetgateway"]
+}
```
Comment (pasipa2) [inline natgateway/terragrunt.hcl:7]: blank line

File: persistent-eip/example-elastic-ip-outputs.tf [hunk 4]
```diff
@@ -0,0 +1,8 @@
+//# outputs
+//
+//output "delius_ndelius_az1_lb_eip" {
+//  value = {
+//    allocation_id = aws_eip.delius_ndelius_az1_lb.id
+//    public_ip     = aws_eip.delius_ndelius_az1_lb.public_ip
+//  }
+//}
```
Comment (pasipa2) [inline persistent-eip/example-elastic-ip-outputs.tf:8]: blank line

File: persistent-eip/example-elastic-ip.tf [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+//      "Name" = "${var.environment_name}-delius-ndelius-az1-lb"
+//    },
+//    {
+//      "Do-Not-Delete" = "true"
+//    },
+//  )
+//  lifecycle {
+//    prevent_destroy = true
+//  }
+//}
```
Comment (pasipa2) [inline persistent-eip/example-elastic-ip.tf:15]: blank line

File: persistent-eip/terragrunt.hcl [hunk 6]
```diff
@@ -0,0 +1,7 @@
+include {
+  path = find_in_parent_folders()
+}
+
+dependencies {
+  paths = []
+}
```
Comment (pasipa2) [inline persistent-eip/terragrunt.hcl:7]: blank line

File: s3/backups/terragrunt.hcl [hunk 7]
```diff
@@ -0,0 +1,7 @@
+include {
+  path = find_in_parent_folders()
+}
+
+dependencies {
+  paths = []
+}
```
Comment (pasipa2) [inline s3/backups/terragrunt.hcl:7]: blank line

File: vpc/outputs-vpc.tf [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+      az2 = module.private_subnet_az2.subnet_cidr,
+      az3 = module.private_subnet_az3.subnet_cidr,
+    },
+    db = {
+      az1 = module.db_subnet_az1.subnet_cidr,
+      az2 = module.db_subnet_az2.subnet_cidr,
+      az3 = module.db_subnet_az3.subnet_cidr,
+    }
+  }
+}
```
Comment (pasipa2) [inline vpc/outputs-vpc.tf:76]: blank line

File: .github/workflows/tag-master-branch-on-merge.yaml (hunk 1 above)
Comment (smjmoj) [reply .github/workflows/tag-master-branch-on-merge.yaml:5]: well spotted

**Review by pasipa2 — APPROVED** [review]

**Pull request merged**